torchao/quantization/quant_api.py

```diff
@@ -857,6 +857,7 @@ def int8_dynamic_activation_int8_weight(
     layout=PlainLayout(),
     act_mapping_type=MappingType.SYMMETRIC,
     weight_only_decode=False,
+    weight_zp_domain=ZeroPointDomain.NONE,
```

Comment:
Hi @jerryzh168, I added a new argument `weight_zp_domain` to `int8_dynamic_activation_int8_weight` & made it `ZeroPointDomain.NONE` by default, since weight is being symmetrically quantized here.

Instead of this change, I could also have instead called `to_affine_quantized_intx` with `zero_point_domain=ZeroPointDomain.NONE`, but I didn't do so because in the main branch, the default value of `ZeroPointDomain.INT` is being used here.

Thanks!

Reply:
actually here, I think we can just hardcode the zero_point_domain in when we call `to_affine_aquantized_intx` in this function, if it's always true for int8_act_int8_weight, unless you have a use case that is not doing this?